decidim-core/config/locales/fr.yml

```diff
@@ -1059,7 +1059,7 @@ fr:
       invalid: '%{authentication_keys} ou mot de passe invalide.'
       invited: Vous avez une invitation en attente, acceptez-la pour terminer la création de votre compte.
       last_attempt: Vous avez encore une tentative avant que votre compte soit verrouillé.
-      locked: Ton compte est bloqué.
+      locked: Votre compte est bloqué.
```

Comment:
Hi @armandfardeau 
as we have independent translations management project called Crowdin, we cannot put translations files in other languages in this same PR, either do merge from there.
Could you remove other language .yml translations files and leave only en.yml file changes please?